src/state/stockMarket/stockMarketActions.ts

```diff
@@ -0,0 +1,38 @@
+import { Stock } from '../AppState';
+
+export const LOAD_STOCKS = 'stockMarketReducer/load-stocks';
+export const ADD_STOCKS = 'stockMarketReducer/add-stocks';
+export const BUY_STOCK = 'stockMarketReducer/buy-stock';
+export const SELL_STOCK = 'stockMarketReducer/sell-stock';
+
+export const loadStocks = () => ({
+    type: LOAD_STOCKS
+});
+
+export const addStocks = (stocks: Stock[]): AddStocksAction => ({
```

Comment:
What is `ADD STOCKS` for? Can't you only buy and sell?

Reply:
Required to load data initially

Reply:
and then what is LoadStocks for? 😮